src/Streamly/FileSystem/Handle.hs

```diff
@@ -76,9 +76,9 @@ module Streamly.FileSystem.Handle
     -- encountered.
 
     , read
-    , readWithBufferOf
+    , readWith
```

Comment:
`readBufferOf` should be kept with deprecation. And same for other released functions as well.